L1Trigger/L1TNtuples/test/step1_ntuples_125.py

```diff
@@ -0,0 +1,151 @@
+    splitLevel = cms.untracked.int32(0)
+)
+
+# Additional output definition
+
+# Other statements
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, '123X_mcRun4_realistic_v3', '')
+
+# Path and EndPath definitions
+process.raw2digi_step = cms.Path(process.RawToDigi)
+process.endjob_step = cms.EndPath(process.endOfProcess)
+process.FEVTDEBUGHLToutput_step = cms.EndPath(process.FEVTDEBUGHLToutput)
+
+#from L1Trigger.Configuration.customisePhase2 import addMenuNtuples
+#process = addMenuNtuples(process)
+process.load("L1Trigger.L1TNtuples.l1PhaseIITreeStep1Producer_cfi")
+process.TFileService = cms.Service("TFileService",
+    fileName = cms.string('L1NtuplePhaseII_Step1.root')
+)
+
+
+# Schedule definition
+process.schedule = cms.Schedule(
+#process.raw2digi_step,
+process.runmenutree,process.endjob_step)#,process.FEVTDEBUGHLToutput_step)
+from PhysicsTools.PatAlgos.tools.helpers import associatePatAlgosToolsTask
+associatePatAlgosToolsTask(process)
+
+#Setup FWK for multithreaded
+#process.options.numberOfThreads = 8
+#process.options.numberOfStreams = 0
+#process.options.numberOfConcurrentLuminosityBlocks = 0
+process.options.eventSetup.numberOfConcurrentIOVs = 1
+
+# customisation of the process.
+
+# Automatic addition of the customisation function from SLHCUpgradeSimulations.Configuration.aging
+from SLHCUpgradeSimulations.Configuration.aging import customise_aging_1000 
+
+#call to customisation function customise_aging_1000 imported from SLHCUpgradeSimulations.Configuration.aging
+process = customise_aging_1000(process)
+
+# Automatic addition of the customisation function from L1Trigger.Configuration.customisePhase2TTNoMC
+#from L1Trigger.Configuration.customisePhase2TTNoMC import customisePhase2TTNoMC 
+
+#call to customisation function customisePhase2TTNoMC imported from L1Trigger.Configuration.customisePhase2TTNoMC
+#process = customisePhase2TTNoMC(process)
+
+# Automatic addition of the customisation function from Configuration.DataProcessing.Utils
+from Configuration.DataProcessing.Utils import addMonitoring 
+
+#call to customisation function addMonitoring imported from Configuration.DataProcessing.Utils
+process = addMonitoring(process)
+
+# Automatic addition of the customisation function from L1Trigger.Configuration.customisePhase2FEVTDEBUGHLT
+from L1Trigger.Configuration.customisePhase2FEVTDEBUGHLT import customisePhase2FEVTDEBUGHLT 
+
+#call to customisation function customisePhase2FEVTDEBUGHLT imported from L1Trigger.Configuration.customisePhase2FEVTDEBUGHLT
+process = customisePhase2FEVTDEBUGHLT(process)
+
+# End of customisation functions
+
+
+# Customisation from command line
+
+# Add early deletion of temporary data products to reduce peak memory need
+from Configuration.StandardSequences.earlyDeleteSettings_cff import customiseEarlyDelete
+process = customiseEarlyDelete(process)
+# End adding early deletion
```

Comment:
Is this file intended for CMSSW testing/ unit testing? Or is this a personal testing file?